### ISSUE TITLE
Workers crashing on 1.5.2

### ISSUE THREAD

**Issue by damiantw** [laravel/octane#674]
- Octane Version: 1.5.2
- Laravel Version: 10.6.0
- PHP Version: 8.2.4
- Server & Version: RoadRunner 2.12.3 (Also, Vapor deployment w/ Octane enabled)
- Database Driver & Version: Postgres 15.2

### Description:

The latest Octane release seems to have a regression related to https://github.com/laravel/octane/commit/7fecb1dcc4cdac60273994b2d831c9a9524940ca.

Workers crash after serving an initial request. Workers stop crashing when I take **any** of the following actions.

1. Downgrade to Octane 1.5.1
2. Remove 'session' and 'session.store' from the `octane.warm` configuration
3. Remove `\Laravel\Octane\Listeners\GiveNewApplicationInstanceToSessionManager::class` as a `RequestReceived` listener

I'm having trouble finding any useful error output from my local environment (nothing is written to the Laravel log). However, after deploying 1.5.2 to the Vapor environment our logs filled w/ error messages like the following.

```json
{
  "exception": {
    "class": "Illuminate\\Contracts\\Container\\BindingResolutionException",
    "message": "Target class [config] does not exist.",
    "code": 0,
    "file": "/var/task/vendor/laravel/framework/src/Illuminate/Container/Container.php:916",
    "previous": {
      "class": "ReflectionException",
      "message": "Class \"config\" does not exist",
      "code": -1,
      "file": "/var/task/vendor/laravel/framework/src/Illuminate/Container/Container.php:914"
    }
  }
}
``` 

### Steps To Reproduce:

**Octane 1.5.2**

<img width="566" alt="Screenshot 2023-04-04 at 2 13 18 PM" src="https://user-images.githubusercontent.com/19997758/229881709-5c938c71-647d-4134-a13d-bc9b83700fc6.png">

**Octane 1.5.1**

<img width="932" alt="Screenshot 2023-04-04 at 2 15 44 PM" src="https://user-images.githubusercontent.com/19997758/229882827-c2a414c0-3e52-4dfe-8847-e117b9896998.png">

**Comment by nunomaduro** [comment]
Fixed on `1.5.3`.

**Comment by taylorotwell** [comment]
Also need Laravel framework `10.6.1` FYI.
